### PR TITLE
update ex15 PS and dvc .git file

### PR DESCRIPTION
**Pull request by PaulWessel**
New plot looks great - now in DVC.

### CODE REVIEW

**Comment by Esteban82** [comment]
Is it possible that you deleted the `doc/examples/images` directory?

**Comment by PaulWessel** [comment]
Cannot possibly happen.  All I did was placing the updated es15.ps in the correct directory (doc/examples/images) and then did

dvc add doc/examples/images

and 

dvc push

All seems fine here, but WTF is those messages from the server, @seisman ?

**Comment by Esteban82** [comment]
Just a general question (maybe for seisman). Is it possible to work directly on the cloud with dvc?

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 name: Tests
 
 on:
-  # pull_request:
+  pull_request:
```
Comment (seisman) [inline .github/workflows/tests.yml:7]:
```suggestion
  # pull_request:
```

**Review by seisman — APPROVED** [review]

**Comment by seisman** [comment]
> Cannot possibly happen. All I did was placing the updated es15.ps in the correct directory (doc/examples/images) and then did
> 
> dvc add doc/examples/images
> 
> and
> 
> dvc push
> 
> All seems fine here, but WTF is those messages from the server, @seisman ?

it's more likely a dvc issue.

**Pull request merged**